ctsm_filenames.csv

```diff
@@ -1,12 +1,12 @@
 ﻿,dir_name,old_fname,new_fname,old_string,new_string
 0,/glade/scratch/negins/ctsm_0110/src/main,column_varcon.F90,ctsm_ColumnVarCon.F90,column_varcon,ctsm_ColumnVarCon
 1,/glade/scratch/negins/ctsm_0110/src/main,initGridCellsMod.F90,ctsm_InitGridCells.F90,initGridCellsMod,ctsm_InitGridCells
-2,/glade/scratch/negins/ctsm_0110/src/main,clm_varsur.F90,ctsm_VarSur.F90,clm_varsur,ctsm_VarSur
+2,/glade/scratch/negins/ctsm_0110/src/main,clm_varsur.F90,ctsm_VarSur.F90,clm_instur,ctsm_VarSur
 3,/glade/scratch/negins/ctsm_0110/src/main,clm_varpar.F90,ctsm_VarPar.F90,clm_varpar,ctsm_VarPar
 4,/glade/scratch/negins/ctsm_0110/src/main,filterColMod.F90,ctsm_FilterCol.F90,filterColMod,ctsm_FilterCol
 5,/glade/scratch/negins/ctsm_0110/src/main,GetGlobalValuesMod.F90,ctsm_GetGlobalValues.F90,GetGlobalValuesMod,ctsm_GetGlobalValues
 6,/glade/scratch/negins/ctsm_0110/src/main,restFileMod.F90,ctsm_RestFile.F90,restFileMod,ctsm_RestFile
-7,/glade/scratch/negins/ctsm_0110/src/main,accumulMod.F90,ctsm_Accumul.F90,accumulMod,ctsm_Accumul
+7,/glade/scratch/negins/ctsm_0110/src/main,accumulMod.F90,ctsm_Accumulators.F90,accumulMod,ctsm_Accumulators

```

Comment:
Here and in a few other places, I felt like a slightly longer name seemed more appropriate with the dropping of 'Mod'. In some cases, a plural form felt more appropriate.

Reply:
Perfect! I agree with the longer names.